source/issues-and-help.txt

```diff
@@ -0,0 +1,39 @@
+=============
+Issues & Help
+=============
+
+.. default-domain:: mongodb
+
+We are lucky to have a vibrant community of {+driver-long+}
+users. We recommend seeking support for general
+questions through the `MongoDB Community Forums
+<https://www.mongodb.com/community/forums>`__.
+
+Bugs / Feature Requests
+-----------------------
+
+To report a bug or to request a new feature in the {+driver-short+},
+please open an issue in JIRA, our issue management tool, using the
+following steps:
+
+1. `Create a JIRA account <https://jira.mongodb.org>`__.
+#. Navigate to `the CSHARP Driver project <https://jira.mongodb.org/browse/CSHARP>`_.
+#. Click :guilabel:`Create Issue`. Please provide as much information as possible about the
+   issue and the steps to reproduce it.
+
+Bug reports in JIRA for the CSHARP Driver project can be viewed by everyone.
+
+If you identify a security vulnerability in the driver or in any other
+MongoDB project, please report it according to the instructions found in
+:manual:`Create a Vulnerability Report </tutorial/create-a-vulnerability-report>`.
+
+Pull Requests
+-------------
+
+We are happy to accept contributions to help improve the driver. We will
+guide user contributions to ensure they meet the standards of the 
+codebase.
+
+To contribute to the official driver, follow the `development and testing guidelines
+<https://github.com/mongodb/mongo-csharp-driver/blob/master/CONTRIBUTING.md>`__
+to ensure your code meets the quality standards.
```

Comment:
Suggestion: 
Rearrange the sentences so they're chronological. That could also help reduce repetition.

```suggestion
We are happy to accept contributions to help improve the driver.
To contribute to the official driver, follow the `development and testing guidelines
<https://github.com/mongodb/mongo-csharp-driver/blob/master/CONTRIBUTING.md>`__
We will guide user contributions to ensure they meet the standards of the 
codebase.
```